tests/push.bats

```diff
@@ -8,6 +8,10 @@ load '../lib/shared'
 # export BUILDKITE_AGENT_STUB_DEBUG=/dev/tty
 # export BATS_MOCK_TMPDIR=$PWD
 
+setup_file() {
+  export BUILDKITE_PLUGIN_DOCKER_COMPOSE_RUN_LABELS="false"
+}
+
```

Comment:
while appreciated, this is not actually needed as the behaviour being tested here does not change :)